src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md

```diff
@@ -11,6 +11,12 @@
   [RS0026](https://github.com/dotnet/roslyn-analyzers/blob/main/src/PublicApiAnalyzers/Microsoft.CodeAnalysis.PublicApiAnalyzers.md#rs0026-do-not-add-multiple-public-overloads-with-optional-parameters).
   ([#1218](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1218))
 
+* Fix the issue of running into the `ArgumentException`: `An instance of
+  EventSource with Guid edc24920-e004-40f6-a8e1-0e6e48f39d84 already exists.`
+  when using multiple instances of `GenevaMetricExporter` by updating
+  `MetricEtwDataTransport` to be a singleton.
```

Comment:
I suggest that we remove "by updating ..." to make it concise (my take is that the reader of the changelog would want to focus on "what does it mean to me" rather than "how you've implemented it", and for folks who have curiosity to know the details, they can always go to the PR - which is why we put the PR link).

I don't feel strongly about this though, so not a blocker.

Reply:
> my take is that the reader of the changelog would want to focus on "what does it mean to me" rather than "how you've implemented it", and for folks who have curiosity to know the details, they can always go to the PR - which is why we put the PR link

I think CHANGELOG could be meant for developers as well? For example, look at this entry in the collector [CHANGELOG](https://github.com/open-telemetry/opentelemetry-collector/blob/main/CHANGELOG.md#v0790) under v0.79.0:

- batchprocessor: Change multiBatcher to use sync.Map, avoid global lock on fast path (#7714)

This is too much detail for users but it gives a good reference point for developers when they quickly want to identify which version and above have that change.